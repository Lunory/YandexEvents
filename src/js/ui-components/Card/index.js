export { default as SmallCard } from './SmallCard'
export { default as MediumCard } from './MediumCard'
export { default as LargeCard } from './LargeCard'
