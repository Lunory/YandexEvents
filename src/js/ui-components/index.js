export { default as Avatar } from './Avatar'
export { default as Button } from './Button'
export { default as FloatingButton } from './FloatingButton'
export { default as Carousel } from './Carousel'
export { default as Image } from './Image'
export { default as Slider } from './Slider'
export { default as Container } from './Container'
export { default as Tabs } from './Tabs'
export { default as TopBar } from './TopBar'
export { default as Icon } from './Icon'
export { default as Card } from './Card'
export { default as SlideCard } from './SlideCard'
export { default as Spinner } from './Spinner'
export { default as MapCard } from './MapCard'
export { default as HolidayCard } from './HolidayCard'
export { default as BottomNavigation } from './BottomNavigation'
