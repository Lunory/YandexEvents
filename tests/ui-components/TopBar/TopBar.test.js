import React from 'react'
import renderer from 'react-test-renderer'

import { TopBar } from 'ui-components'

describe('TopBar', () => {
  it('Проверка снепшота компонента', () => {
    const component = renderer.create(
      <TopBar />
    )

    expect(component.toJSON()).toMatchSnapshot()
  })
})
