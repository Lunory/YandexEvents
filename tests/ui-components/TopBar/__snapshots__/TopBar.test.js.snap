// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopBar Проверка снепшота компонента 1`] = `
<div
  className=""
>
  <button
    className=""
    onClick={undefined}
    style={
      Object {
        "left": 0,
        "top": 24,
      }
    }
  />
  <h2
    className=""
  />
  <button
    className=""
    onClick={undefined}
    style={
      Object {
        "right": 56,
        "top": 24,
      }
    }
  />
  <button
    className=""
    onClick={undefined}
    style={
      Object {
        "right": 0,
        "top": 24,
      }
    }
  />
</div>
`;
